subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -55,6 +55,7 @@ static int init_reset(void);
 #if defined(CONFIG_BT_PERIPHERAL)
 static void ticker_update_latency_cancel_op_cb(uint32_t ticker_status,
 					       void *params);
+static void peripheral_latency_cancel(struct ll_conn *conn, uint16_t handle);
```

Comment:
Is latency already off during encryption setup, or would it be needed for ll_start_enc_req_send as well?

Reply:
No, as the encryption setup and key refresh procedure is central initiated, a spec compliant Host should use latency set to zero. BT Spec. Version 5.2 | Vol 3, Part C, Section 9.3.12.2.